husky2/plugin.yaml

```diff
@@ -0,0 +1,252 @@
+# Metadata for the plugin
+plugin:
+    # Global plugin attributes
+    type: interface                   # plugin type (gateway, interface, protocol, system, web)
+    description:
+        de: 'Husqvarna Automower Plugin'
+        en: 'Husqvarna automower plugin'
+    maintainer: <autor>
```

Comment:
maybe you can add here  a support thread, a link to the documentation for user_doc and a maintainer?

Reply:
No, I used only the pictures wich where already in the old husky plugin. Would it be enough if I make a watermark or something on them?